src/main/kotlin/com/rose/gateway/minecraft/logging/LogFormatter.kt

```diff
@@ -0,0 +1,23 @@
+package com.rose.gateway.minecraft.logging
+
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.util.logging.LogRecord
+import java.util.logging.SimpleFormatter
+
+/**
+ * This class determines the format of the FileLogger
+ */
+class LogFormatter : SimpleFormatter() {
+
+    override fun format(record: LogRecord): String? {
+        val zdt = ZonedDateTime.ofInstant(
+            record.instant, ZoneId.systemDefault()
+        )
+        val message = formatMessage(record)
+
+        return "${record.level.localizedName}: $message " +
```

Comment:
This string could be made to look a bit nicer if you move record.level.localizedName into a variable and then just used that variable name for interpolation. this especially goes for the formatted date time below. This will also allow for the entire string to fit on one line.